.github/workflows/firesim-run-tests.yml

```diff
@@ -14,6 +14,7 @@ env:
   AWS-SECRET-ACCESS-KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
   AWS-DEFAULT-REGION: ${{ secrets.AWS_DEFAULT_REGION }}
   FIRESIM_PEM: ${{ secrets.FIRESIM_PEM }}
+  MANAGER_FIRESIM_LOCATION: "~/firesim"
```

Comment:
I assume this being `/home/centos` creates problems w/ non-Centos based environments - what do you do on Azure? IIRC in the past there were some problems with having `~` or `$HOME` in scripts (though I can't find an example of it). If this is working, I'm fine with it.

Reply:
We don't run on azure yet. I can switch it but this does seem to work fine.

Reply:
This is fine with me.